pcstac/requirements.txt

```diff
@@ -249,7 +249,7 @@ stac-pydantic==2.0.3
     #   stac-fastapi-extensions
     #   stac-fastapi-pgstac
     #   stac-fastapi-types
-starlette==0.22.0
+starlette==0.25.0
```

Comment:
We can't do that because of https://github.com/microsoft/planetary-computer-apis/blob/90cd8463f3484ce087710ab22ec90df374e7d5a9/pccommon/setup.py#L7-L8